src/stories/Quiz/Hooks/index.tsx

```diff
@@ -223,7 +223,7 @@ function HooksTemplate(quizConfiguration) {
               </div>
               <button {...getResetQuizButtonProps('secondary')}>
```

Comment:
Can we add a parameter to getResetQuizButtonProps so it takes CTA text to be custom?